#File generated by Pentaho Translator for package 'org.pentaho.di.core.database' in locale 'ko_KR'
#
#
#Sun Jun 01 22:07:30 KST 2008
Database.UnableToLoadConnectionPoolDriver.Exception=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uC5F0\uACB0 \uD480(pool) ''{0}'' \uC640 \uD074\uB798\uC2A4 [{1}]\uB97C \uB85C\uB529\uD560 \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
Database.CreatingConnectionPool=''{0}''\uB97C \uC704\uD55C \uB370\uC774\uD14C\uBCA0\uC774\uC2A4 \uC5F0\uACB0 \uD480(pool) \uC0DD\uC131 \uC911...
Database.Exception.UnableToSetSavepointName=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uD2B8\uB79C\uC7AD\uC158 savepoint ''{0}'' \uB97C \uC124\uC815\uD560 \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
DatabaseMeta.Error.UnexpectedError=\uC2A4\uD0A4\uB9C8 \uC815\uBCF4\uB97C \uAC00\uC838\uC624\uB294 \uC911 \uC54C \uC218 \uC5C6\uB294 \uC624\uB958
DatabaseMeta.Error.UnexpectedCatalogError=\uCE74\uD0C8\uB85C\uADF8 \uC815\uBCF4\uB97C \uAC00\uC838\uC624\uB294 \uC911 \uC54C \uC218 \uC5C6\uB294 \uC624\uB958
DatabaseMeta.Info.ClosingDbConnection=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uC5F0\uACB0\uC744 \uB2EB\uB294 \uC911
DatabaseMeta.report.ConnectionOk=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 [{0}] \uC5F0\uACB0 \uC131\uACF5
Database.UnableToCheckIfConnectionPoolExists.Exception=Error checking if the connection pool is registered
DatabaseMeta.report.ConnectionWithPartOk=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 [{0}] \uC758 \uD30C\uD2F0\uC158 \uC544\uC774\uB514 [{1}] \uC5F0\uACB0 \uC131\uACF5
Database.Exception.UnableToReleaseSavepoint=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uD2B8\uB79C\uC7AD\uC158 savepoint\uB97C release \uD560 \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
Database.Exception.UnableToRollbackToSavepoint=savepoint\uAE4C\uC9C0 \uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uD2B8\uB79C\uC7AD\uC158\uC744 \uB864\uBC31\uD560 \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
DatabaseMeta.report.ConnectionError=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 [{0}] \uC5F0\uACB0  \uC624\uB958 \:
DatabaseMeta.Info.ConnectingDb=\uB370\uC774\uD130\uBCA0\uC774\uC2A4\uC5D0 \uC5F0\uACB0 \uC911
DatabaseMeta.report.Port=\uD3EC\uD2B8           \: 
Database.Exception.UnableToSetSavepoint=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uD2B8\uB79C\uC7AD\uC158 svaepoint\uB97C \uC124\uC815\uD560 \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
DatabaseMeta.report.Hostname=\uD638\uC2A4\uD2B8      \: 
DatabaseMeta.Info.GettingTables=\uD14C\uC774\uBE14\uC744 \uAC00\uC838\uC624\uB294 \uC911
DatabaseUtil.DSNotFound=\uC798\uBABB\uB41C \uB370\uC774\uD130 \uC18C\uC2A4\:''{0}''
DatabaseMeta.Info.GettingViews=\uBDF0\uB97C \uAC00\uC838\uC624\uB294 \uC911
DatabaseMeta.Info.GettingSynonyms=\uB3D9\uC758\uC5B4 \uAC00\uC838\uC624\uB294 \uC911
DatabaseMeta.Info.GettingInfo=\uCE74\uD0C8\uB85C\uADF8 \uC815\uBCF4 \uAC00\uC838\uC624\uB294 \uC911
Database.UnableToPreLoadConnectionToConnectionPool.Exception=Unable to pre-load connection to the connection pool
DatabaseMeta.report.DatabaseName=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uC774\uB984  \: 
BaseDatabaseMeta.TestConnectionReportNotImplemented.Message=I''m sorry, we can''t create a test report for this database.
DatabaseMeta.Error.UnableRetrieveDbInfo=\uC624\uB958\uB85C\uC778\uD574 \uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uC815\uBCF4\uB97C \uAC00\uC838\uC62C \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
DatabaseMeta.Info.GettingMetaData=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uBA54\uD0C0\uB370\uC774\uD130\uB97C \uAC00\uC838\uC624\uB294 \uC911
Database.Exception.UnableToDisableAutoCommit=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uC5F0\uACB0 ''{0}'' \uC5D0\uC11C auto-commit\uC744 \uBE44\uD65C\uC131\uD654 \uD560 \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
DatabaseMeta.report.ConnectionWithPartError=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 [{0}], \uD30C\uD2F0\uC158 \uC544\uC774\uB514 [{1}] \: {2} \uC5F0\uACB0 \uC624\uB958
Database.CreatedConnectionPool=''{0}''\uB97C \uC704\uD55C \uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uCEE4\uB125\uC158 \uD480(pool)\uC744 \uC0DD\uC131\uD558\uC600\uC2B5\uB2C8\uB2E4.
DatabaseMeta.Info.GettingInfoFromDb=\uB370\uC774\uD130\uBCA0\uC774\uC2A4\uC5D0\uC11C \uC815\uBCF4\uB97C \uAC00\uC838\uC624\uB294 \uC911...
Database.Exception.UnableToEnableAutoCommit=\uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uC5F0\uACB0 ''{0}'' \uC5D0\uC11C auto-commit\uC744 \uD65C\uC131\uD654 \uD560 \uC218 \uC5C6\uC2B5\uB2C8\uB2E4.
DatabaseMeta.Info.GettingSchemaInfo=\uC2A4\uD0A4\uB9C8 \uC815\uBCF4\uB97C \uAC00\uC838\uC624\uB294 \uC911
